File: app/src/main/java/com/chiron/playpalyer/media/IDecoderProgress.java
package com.chiron.playpalyer.media;

/**
 * 解码进度
 */
interface IDecoderProgress {
    /**
     * 视频宽高变化时的回调
     */
    void videoSizeChange(int width,int height, int rotationAngle);

    /**
     * 视频进度变化时的回调
     */
    void videoProgressChange(long pos);
}

File: app/src/main/java/com/chiron/playpalyer/mediac/IDecoderProgress.kt
package com.chiron.playpalyer.mediac

/**
 * 解码进度
 */
interface IDecoderProgress {
    /**
     * 视频宽高变化时的回调
     */
    fun videoSizeChange(width: Int, height: Int, rotationAngle: Int)

    /**
     * 视频进度变化时的回调
     */
    fun videoProgressChange(pos: Long)
}